src/evict/evict_lru.c

```diff
@@ -8,7 +8,7 @@
 
 #include "wt_internal.h"
 
-static int __evict_clear_all_walks_and_saved_tree(WT_SESSION_IMPL *);
+static int __evict_clear_all_walks_and_saved_tree(WT_SESSION_IMPL *, bool);
```

Comment:
I guess you are anticipating further usages of this function? I don't think we need the bool at this stage with only one caller.

Reply:
There are currently two callers: one with "true", another with "false".

Reply:
There's only one caller here. I am 100% sure about that.